extensions/skins/conversation_v1/static/conversation.css

```diff
@@ -124,6 +129,46 @@ html, body {
   max-width: 1084px;
 }
 
+.conversation-skin-help-card-container.ng-enter,
+.conversation-skin-help-card-container.ng-leave {
```

Comment:
Turns out this needs to be .ng-leave-active, the card isn't actually animating on exit at the moment.